frontend/src/pages/user.tsx

```diff
@@ -1,12 +1,21 @@
 import { Stack, Title } from '@mantine/core';
+import { useTranslation } from 'next-i18next';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
 
 import UserForm from '../components/forms/user';
 import { checkForAuthError, getTournaments, getUser } from '../services/adapter';
 import { getLogin } from '../services/local_storage';
 import Layout from './_layout';
 
+export const getServerSideProps = async ({ locale }: { locale: string }) => ({
+  props: {
+    ...(await serverSideTranslations(locale, ['common'])),
+  },
+});
```

Comment:
It's a bit annoying maybe that this has to be defined in every page file, but we can look into it later.